libraries/lib-stretching-sequence/readme.md

```diff
@@ -0,0 +1,76 @@
+# `StretchingSequence`
```

Comment:
If you really want to include this in the tree, the CMakeLists.txt comment can refer the reader to it for more explanation.

Reply:
I'm skipping the rest of this big commit for now, I will return and review it in detail last

Reply:
I have to update this, thanks for reminding.